docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
```

Comment:
could you please add this service to the main docker compose file as we already have one with all the services under the docker package?